src/sentry/identity/vsts/provider.py

```diff
@@ -1,16 +1,26 @@
 from __future__ import absolute_import
 
-from django import forms
+from sentry import http, options
 
-from sentry import http
-from sentry import options
-
-from sentry.web.helpers import render_to_response
 from sentry.identity.oauth2 import OAuth2Provider, OAuth2LoginView, OAuth2CallbackView
-from sentry.pipeline import PipelineView
 from sentry.utils.http import absolute_uri
 
 
+def get_user_info(access_token):
+    session = http.build_session()
+    resp = session.get(
+        'https://app.vssps.visualstudio.com/_apis/profile/profiles/me?api-version=1.0',
```

Comment:
Is api version 1.0 going to be supported? The current version for most vsts calls is 4.1. Is this a part of another api? What's the current version?

Reply:
quoting the msft PM "yes, it’s safe. You’ll probably get the same results/behavior calling this API with either 1.0, 2.0, … 5.0-preview (this API hasn’t changed since it was originally introduced, but we always increment the version of the full API set at the same time)"